src/test/stepFunctions/commands/visualizeStateMachine.test.ts

```diff
@@ -23,55 +24,6 @@ const mockGlobalStorage: vscode.Memento = {
     get: sinon.stub().returns(undefined),
 }
 
-const mockUriOne = vscode.Uri.file('uri1')
-
-const mockTextDocumentOne: vscode.TextDocument = {
```

Comment:
Step functions visualization test was causing segfaults in VS Code (min ver) due to the use of 'mock' text documents. The tests also don't try to test much to begin with. More than half the file is dedicated to testing if the manager caches webviews. But that's something that should be generalized and tested in isolation.